docs/zedtokens-and-zookies.md

```diff
@@ -0,0 +1,187 @@
+# ZedTokens and Zookies
```

Comment:
Rather than the focus of this being documenting zed tokens / zookies, could we make this focused on "consistency and availability of spicedb" in general? Then we can answer questions like

- where does SpiceDB fit into the CAP theorem?
- what knobs for consistency are available to application developers?
- what happens if SpiceDB is unavailable?

and also cover zed tokens/zookies in a way that people will discover them when they aren't familiar with the terms or how they relate to the system in general

Reply:
IMO most people will not care initially about reading about the CAP theorem, but rather how to effectively use the system. The consistency knobs are described in the other consistency document.

If we want a section on CAP guarantees, we can add that, but it is likely too technical for people trying to integrate effectively initially.